recipes/hy/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "hy" %}
+{% set version = "0.11.1" %}
+{% set sha256 = "ba33addb81077c96b53f3353a8f296bcb5c969d51b935fc7c922fc78c8d958d7" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - flake8
+    - coverage
+    - requests
+    - clint
+    - rply
+    - astor
+  run:
+    - python
+
+test:
+  imports:
+    - hy
+
+about:
+  home: https://github.com/hylang/hy
+  license: MIT
```

Comment:
Ideally we would add `license_file: LICENSE`. However, that is not currently possible as the license is not included in the `sdist`. Though this could be possible in the future if the license file is added to the manifest. Please see PR ( https://github.com/hylang/hy/pull/1186 ) for how this would be done.